internal/builtinfunctions/functions.go

```diff
@@ -550,13 +550,11 @@ func getBindConstantsFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("Bind Constants"),
 			schema.PointerTo(
-				"Returns a list of objects each containing two properties: an item property which contains "+
-					"the corresponding item from the list in the first parameter; and, a constant property which "+
-					"contains the value of the second input parameter. "+
-					fmt.Sprintf("The two names are joined with the delimiter %q. ", CombinedObjIDDelimiter)+
-					fmt.Sprintf("Each object has two properties `%s` and `%s`.\n", CombinedObjPropertyItemName, CombinedObjPropertyConstantName)+
+				"Returns a list of objects each containing two properties. "+
 					fmt.Sprintf("Param 1: Value(s) to be included in the `%s` field \n", CombinedObjPropertyItemName)+
-					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item", CombinedObjPropertyConstantName)),
+					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item\n", CombinedObjPropertyConstantName)+
+					"The output list items will have an Object ID and schema name as documented in "+
+					"https://arcalot.io/arcaflow/workflows/schemas/#generated-combined-schema-names."),
```

Comment:
There's a detail here which has been bugging me for weeks now:  `bindConstants()` returns a value whose schema is named according to the types of the two parameters -- e.g., `string__integer` -- even though the value returned is _actually a **list**_.

So, the name of the schema _for the return value_ is actually the same as the name of the schema for the _items_ in the returned list...right??

So, when we say
> The output list items will have an Object ID and schema name as documented

that's true, but I think what we want to be describing is the ID and name of the returned value, not its list items.  No?

Reply:
No, that's not what we want since that is not the Object ID that user would need to complete their workflow.